hi-ml-azure/conftest.py

```diff
@@ -50,3 +50,4 @@ def random_folder() -> Generator:
     remove_and_create_folder(folder)
     print(f"Created temporary folder for test: {folder}")
     yield folder
+    shutil.rmtree(folder)
```

Comment:
I would love to leave that in place. I find it very helpful to have all results still in the results folder - they will anyway be cleaned up completely at the start of the next test run.